common/lib/xmodule/xmodule/js/src/video/07_video_volume_control.js

```diff
@@ -310,6 +342,15 @@ function() {
 
                     this.decreaseVolume();
                     return false;
+
+                case KEY.SPACE:
+                case KEY.ENTER:
+                    if (event.shiftKey) {
```

Comment:
They keydown/keyup cases have this comment:
```
 // Shift + Arrows keyboard shortcut might be used by
 // screen readers. In this case, do nothing.
```

Is the `event.shiftKey` conditional on the space/enter case for similar reasons?  A similar comment would make this a bit more clear.